app/lib/screens/history/history.dart

```diff
@@ -1,27 +1,29 @@
 import 'package:flutter/material.dart';
 
-class TravelHistory extends StatefulWidget {
-  @override
-  _TravelHistoryState createState() => _TravelHistoryState();
-}
-
-class _TravelHistoryState extends State<TravelHistory> {
-  void initState() {
-    super.initState();
-    print('Init: Travel history');
-  }
-
+class TravelHistory extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    print('Building.... Travel History page');
     return Scaffold(
-      appBar: AppBar(
-        title: Text('Back to Home'),
-      ),
-      body: Center(
-        child: Text(
-          'This is Travel History page',
-          style: Theme.of(context).textTheme.headline5,
-        ),
+      body: ListView.builder(
+        itemCount: 20,
+        itemBuilder: (context, index) {
+          return Card(
+            color: index % 2 == 0 ? Colors.grey[300] : null,
```

Comment:
Haha, alternating colours trick I see

Reply:
I am pretty sure I learned this in cs 110 haha

Reply:
I'd prefer to tell the table this in language than use a hacky trick. Is this the only way to do this as described by community?

Reply:
Most likely, I just did this for fun and part of experimentation